src/components/Chat/Playback/PlaybackControls.tsx

```diff
@@ -67,21 +68,32 @@ export const PlaybackControls = ({
     );
   }, [activeIndex, isActiveIndex, selectedConversations]);
 
-  const activeMessageContent = useMemo(() => {
-    if (
-      isActiveIndex &&
-      isNextMessageInStack &&
-      selectedConversations.length &&
-      selectedConversations[0].playback &&
-      selectedConversations[0].playback.messagesStack[activeIndex].content
-    ) {
-      return selectedConversations[0].playback.messagesStack[activeIndex]
-        .content;
+  const activeMessage = useMemo(() => {
+    if (!isActiveIndex) {
+      return;
     }
-
-    return '';
+    const CURRENT_PLAYBACK = selectedConversations[0]?.playback;
```

Comment:
I suggest to use UPPERCASE only for global constants which are reusable across multiple files and same between rerenders

Reply:
Thank you for suggestion.
Changed to camelCase.